sdk/rust-examples/sort-numbers/src/main.rs

```diff
@@ -0,0 +1,44 @@
+//! Sorting Numbers Example
+//!
+//! ## Context
+//!
+//! Read an input of unsorted_numbers.txt, sort them and write to sorted_numbers.txt
+//!
+//! ## Authors
+//!
+//! The Veracruz Development Team.
+//!
+//! ## Copyright
+//!
+//! See the file `LICENSING.markdown` in the Veracruz root directory for licensing and
+//! copyright information.
+
+
+use std::fs;
+
+fn main() {
+    let file_vec = match fs::read("/input/unsorted_numbers.txt") {
+        Ok(file_vec) => file_vec,
```

Comment:
A nicer way of doing this would be:

```rust
let file_vec = fs::read("input/unsorted_numbers.txt").map_err(|e| eprintln!(e))?;
```

...and changing the return type of `main` to return an appropriate error type.  For example:

```rust
use std::{fs, io};

fn main() -> Result<(), io::Error> {
    ...
}
```